src/main/java/inventorysetups/InventorySetupsPlugin.java

```diff
@@ -1495,7 +1495,7 @@ public void updateSlotFromSearch(final InventorySetupsSlot slot, boolean allowSt
 								panel.getCurrentSelectedSetup().getAdditionalFilteredItems();
 						if (!additionalFilteredItemsHasItem(finalId, additionalFilteredItems))
 						{
-							removeAdditionalFilteredItem(slot, additionalFilteredItems);
+							removeAdditionalFilteredItem(slot);
 							addAdditionalFilteredItem(finalId, slot.getParentSetup(), additionalFilteredItems);
 						}
 						return;
```

Comment:
I'm not sure why this uses panel.setup.afi instead of slot.parent.afi  If needed I can modify the changes so this functions as written again.